yugabyte/src/yugabyte/ysql/client.clj

```diff
@@ -32,6 +32,49 @@
    :connectTimeout (/ default-timeout 1000)
    :socketTimeout  (/ default-timeout 1000)})
 
+(defn query
+  "Like jdbc query, but includes a default timeout in ms.
+  Requires query to be wrapped in a vector."
+  [conn sql-params]
+  (j/query conn sql-params {:timeout default-timeout}))
+
+(defn insert!
+  "Like jdbc insert!, but includes a default timeout."
+  [conn table values]
+  (j/insert! conn table values {:timeout default-timeout}))
+
+(defn update!
+  "Like jdbc update!, but includes a default timeout."
+  [conn table values where]
+  (j/update! conn table values where {:timeout default-timeout}))
+
+(defn execute!
+  "Like jdbc execute!!, but includes a default timeout."
+  [conn sql-params]
+  (j/execute! conn sql-params {:timeout default-timeout}))
+
+(defn select-first-row
+  "Selects a first row from table with a WHERE-clause, returning nil if no rows were found"
+  [conn table-name where-clause]
+  (let [query-string (str "SELECT * FROM " table-name " WHERE " where-clause)
+        query-res    (query conn query-string)
+        res          (first query-res)]
```

Comment:
Can we use something like `LIMIT 1` in query instead of doing it on client side?

Reply:
We can add that optimization, sure - but the rest of the code won't change, it would just speed up execution. Did that.